investor_wallet_platform_base/models/operation_request.py

```diff
@@ -18,26 +18,27 @@ def default_structure(self):
         default=default_structure,
     )
 
-    def get_share_trans_mail_template(self):
-        templ_obj = self.env["mail.template"]
-        return templ_obj.get_email_template_by_key(
+    def _get_share_transfert_mail_template(self):
+        return self.env["mail.template"].get_email_template_by_key(
             "certificate_trans", self.structure
         )
 
-    def get_share_update_mail_template(self):
+    def _get_share_update_mail_template(self):
```

Comment:
some code uses `…_mail_…` while other `…_email_…`. it would be better to use only one naming. i don’t know which one is more odooesque though.

Reply:
Agreed 👌 I would think `mail` is the best since the model name is `mail.mail`.